brave/brave-ios
# Fix #6428: Multichain assets in Account Details

## Summary of Changes
- Display assets from all networks (for the account coin type, ex. Ethereum / Solana) in Account Details
- Created `AccountActivityStoreTests` and added unit tests for both Ethereum account and Solana account

This pull request fixes #6428

## Submitter Checklist:


## Test Plan:
1. Open 'Accounts' tab and tap any Ethereum account to open account details
2. Verify assets from all Ethereum networks are displayed with correct balances & price
3. Verify transactions are still displayed (note: only selected network transactions are displayed still, this matches desktop behaviour).
4. Tap 'back' to go back to accounts list
5. Tap on any Solana account to open account details for that account
6. Verify assets from all Solana networks are displayed with correct balances and price
7. Verify transactions are still displayed (note: only selected network transactions are displayed still, this matches desktop behaviour).


## Screenshots:

https://user-images.githubusercontent.com/5314553/204024214-a1089771-8669-46b6-81bc-71c55b7289a2.mp4



## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
